### DISCUSSION TITLE
FlutterFlow and the multi-select field

### DISCUSSION THREAD

**Discussion by JacquesBGit** [AntlerVC/firetable#785] [Support / Q&A] [answered]
Multi-selects fields don't seem to work in FlutterFlow, right ?
As soon as I import data from a multi-select field from ROWY, the "document" disapears from the FlutterFlow UI. Even when I clear all the data from Rowy, two bracets remain... see image ... and the rows won't come back in FlutterFlow...
Any Idea ?
<img width="258" alt="Capture d’écran 2022-09-07 à 20 53 52" src="https://user-images.githubusercontent.com/7407513/189009999-a6b82947-4367-4be9-9ff9-587e9e64cfb2.png">

**Comment by ghfsystem** [comment, marked as answer]
I don't think there should be a problem. are you on FF discord group? You can reach me there. I am generally active under the name Dahobi
